systemd/ripple.service

```diff
@@ -4,8 +4,8 @@ After=wpeframework.service
 
 [Service]
 Type=notify
-Environment="RUST_LOG=debug"
```

Comment:
IMHO, this should still be here as a fallback of last resort, as `Environment="RUST_LOG=info"`

Reply:
Would System Services set the Environment values before ExecStartPre

Reply:
Seems like it , I was wondering if we needed to keep this as a fallback in case something happens in `ExecStartPre`.